qemu/tests/fio_windows.py

```diff
@@ -33,45 +31,35 @@ def run(test, params, env):
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
-    session_sys = vm.wait_for_login(timeout=timeout)
-    session_data = vm.wait_for_login(timeout=timeout)
+    session = vm.wait_for_login(timeout=timeout)
 
     error_context.context("Format disk", logging.info)
-    utils_misc.format_windows_disk(session_sys, params["disk_index"],
+    utils_misc.format_windows_disk(session, params["disk_index"],
                                    mountpoint=params["disk_letter"])
     try:
-        installed = session_sys.cmd_status(check_installed_cmd) == 0
+        installed = session.cmd_status(check_installed_cmd) == 0
         if not installed:
-            dst = r"%s:\\" % utils_misc.get_winutils_vol(session_sys)
+            dst = r"%s:\\" % utils_misc.get_winutils_vol(session)
 
             error_context.context("Install fio in guest", logging.info)
             install_cmd = params["install_cmd"]
             install_cmd = re.sub(r"DRIVE:\\+", dst, install_cmd)
-            session_sys.cmd(install_cmd, timeout=180)
+            session.cmd(install_cmd, timeout=180)
             time.sleep(30)
             config_cmd = params.get("config_cmd")
             if config_cmd:
-                session_sys.cmd(config_cmd)
+                session.cmd(config_cmd)
 
         error_context.context("Start fio in guest.", logging.info)
-        # FIXME:Here use the timeout=(cmd_timeout*2)
-        # Will determine a better specific calculation later
-        fio_thread_data = utils_misc.InterruptedThread(session_data.cmd_status_output,
-                                                       (fio_cmd_data, (cmd_timeout*2)))
-        fio_thread_data.start()
-        status_sys, output_sys = session_sys.cmd_status_output(fio_cmd_sys,
-                                                               timeout=(cmd_timeout*2))
-        status_data, output_data = fio_thread_data.join()
-        if status_sys or status_data:
-            test.error("Failed to run fio, output: %s\n%s" % (output_sys, output_data))
+        status, output = session.cmd_status_output(fio_cmd, timeout=(cmd_timeout*2))
+        if status:
+            test.error("Failed to run fio, output: %s" % (output))
```

Comment:
delete () for (output) is ok, such as  test.error("Failed to run fio, output: %s" % output)

Reply:
Thanks, has updated~